ql/src/java/org/apache/hadoop/hive/ql/metadata/Hive.java

```diff
@@ -1459,6 +1464,22 @@ public Table getTable(final String dbName, final String tableName, boolean throw
     return new Table(tTable);
   }
 
+  /**
+   * Get ValidWriteIdList for the current transaction.
```

Comment:
I think it would be good to have more details in this javadoc. Specifically, that this fetches the ValidWriteIdList from the metastore for the given table if txnManager has a open transaction.

Reply:
Good idea, added additional details.